manim/animation/creation.py

```diff
@@ -198,6 +307,8 @@ def __init__(self, text, **kwargs):
 
 
 class ShowSubmobjectsOneByOne(ShowIncreasingSubsets):
+    """Show one submobject at a time, removing all previous ones from screen."""
```

Comment:
```suggestion
    """Show one submobject at a time, removing all previously displayed ones from screen."""
```
What do you think about including the word "displayed?"